docs/apicast-crd-reference.md

```diff
@@ -182,7 +182,7 @@ spec:
 |                 `production.env.apicastLogLevel`                  | `string`  |      No      |                `warn`                 |                           Openresty log level                            |
 |               `production.env.apicastOIDCLogLevel`                | `string`  |      No      |               `notice`                |                   OpenID Connect integration log level                   |
 |                   `production.marin3r.enabled`                    | `boolean` |     Yes      |                   -                   |               Enable (`true`) or disable (`false`) marin3r               |
-|                `production.marin3r.annotations.{}`                 |   `map`   |      No      |                   -                   |                        Map of marin3r annotations                        |
+|                `production.marin3r.anotations.{}`                 |   `map`   |      No      |                   -                   |                        Map of marin3r annotations                        |
```

Comment:
Something weid happened here, it was fixed on previous commit, and reverted on that commit

Reply:
I guess a I messed up the rebase. It is fixed now.